e2e/cli-e2e/tests/compare.e2e.test.ts

```diff
@@ -43,10 +45,12 @@ describe('CLI compare', () => {
     await cleanTestFolder('tmp/e2e');
   });
 
-  it('should compare report.json files and create report-diff.json and report-diff.md', async () => {
+  // eslint-disable-next-line vitest/no-disabled-tests
+  it.skip('should compare report.json files and create report-diff.json and report-diff.md', async () => {
```

Comment:
It is unclear to me why this happens. 

See error on my local machine
<img width="1636" alt="Screenshot 2024-08-30 at 21 42 06" src="https://github.com/user-attachments/assets/6d1d1f59-8230-4d6d-a896-1ba4a5f26aaa">

Reply:
I've figured out where the problem is - the test aren't running `@code-pushup/cli` from the local registry :disappointed: The `npx @code-pushup/cli` command installs and executes the latest version from https://registry.npmjs.org/. This explains why the `report-diff.md` output is different - the updated formatting from #793 hasn't been released yet.